src/tests/test_utils.rs

```diff
@@ -35,3 +35,7 @@ where
     let mut contract = EvmContract::default();
     test(&mut contract)
 }
+
+pub fn clear_attached_deposit() {
```

Comment:
Rename function. It doesn't clear the attached deposit, it changes the environment. Maybe call it reset context :thinking:

Reply:
sure. technically the only thing it changes about the environment so far is the attached_value (and technically account balance which doesn't seem to do be doing anything for the tests), but I see what you're saying

Reply:
fiixed